tutorials/rllib_pistonball.py

```diff
@@ -60,6 +71,8 @@ def env_creator(args):
 
 
 if __name__ == "__main__":
+    shutdown()
```

Comment:
It seems odd to put this shutdown at the beginning of the script. Shouldn't it come at the end?

Reply:
indeed, but this shuts any instances of ray that were not closed properly prior to running this script

Reply:
probably isnt required though, however, i commented it out with my last commit, just in-case someone else happens to run into the same problem i was

Reply:
Interesting. Makes sense that this would be helpful, we have had problems with zombie ray processes before.